src/AgentContainer.cpp

```diff
@@ -204,8 +204,6 @@ void AgentContainer::moveAgentsToWork ()
     }
 
     m_at_work = true;
-
-    Redistribute();
```

Comment:
@stevenhofmeyr specifically added the `Redistribute()` in [this PR](https://github.com/AMReX-Codes/ExaEpi/pull/79). Can they really be removed?

Reply:
Hi @stevenhofmeyr, Redistribute() operation is costly so Andrew created a small particle container for travel agents and only redistribute agents in this container. For work and home activities, we have a static communication pattern which can be handled by the arrangement of bins.

Reply:
@tannguyen153 (and @atmyers), I tagged you both on a thread in the ExaEpi channel on the EMERGE Slack - @stevenhofmeyr explained there why he add these `Redistribute()` calls. Can you take a look?